x/burn/internal/keeper/burn.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	accountExported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+)
+
+func (k Keeper) BurnCoins(ctx sdk.Context, acc string) sdk.Error {
+	bunAccount, err := getAccount(ctx, k, acc)
+	if err != nil {
+		return sdk.ErrInvalidAddress(err.Error())
+	}
+
+	if bunAccount == nil {
+		return nil
+	}
+
+	amt := bunAccount.GetCoins()
+	if amt.Empty() {
+		return nil
+	}
+
+	ctx.Logger().Info("find burn coin.", fmt.Sprintf("address: %s, coins: %s", bunAccount.GetAddress(), bunAccount.GetCoins()))
+
+	_, err = k.bankKeeper.SubtractCoins(ctx, bunAccount.GetAddress(), amt)
+
+	if err != nil {
+		return sdk.ErrInvalidCoins(err.Error())
```

Comment:
Because `bankKeeper.SubstractCoins()` already returns `sdk.Error`, I think we can just return it as it is.
```suggestion
		return err
```

Reply:
@youngjoon-lee 
OK. Change sdk.Error to native error